src/Dialect/ONNX/ONNXOps.cpp

```diff
@@ -282,8 +282,20 @@ static LogicalResult processConvStrideParam(
 
   auto stridesOpt = op->strides();
   if (stridesOpt.hasValue()) {
-    if (ArrayAttrSize(stridesOpt) != kernelRank)
-      return op->emitError("strides rank is not the same as the spatial rank");
+    if (ArrayAttrSize(stridesOpt) != kernelRank) {
+      if (ArrayAttrSize(stridesOpt) != 1)
+        return op->emitError("strides rank is not the same as the spatial rank");
+
+      // Stride in Pytorch can be represented by single value. ONNX exportation
```

Comment:
Is this a bug with pytorch onnx export? Why haven't we encountered this before on other graphs?

Reply:
Here is an example of what I see right after export:
```
 %117 = "onnx.Conv"(%114, %115, %116) {dilations = [1, 1], group = 32 : si64, kernel_shape = [3, 3], onnx_node_name = "Conv_111", pads = [0, 0, 0, 0], strides = [2]} : (tensor<*xf32>, tensor<32x1x3x3xf32>, tensor<32xf32>) -> tensor<*xf32>
```
This is the pytorch code for `conv`:
```
self._conv_stem = Conv2d(in_channels, out_channels, kernel_size=3, stride=2, bias=False)
```
It can be recognized as a bug in `onnx` export in my opinion.

Reply:
Yeah you are right. I looked at the ONNX definition again and this seems like a mismatch.